pocs/utils/images.py

```diff
@@ -137,13 +140,14 @@ def get_solve_field(fname, replace=True, remove_extras=True, **kwargs):
         proc.kill()
         raise error.Timeout("Timeout while solving")
     else:
-        if proc.returncode == 3:
-            raise error.SolveError('solve-field not found: {}'.format(output))
-
         if verbose:
+            print("Returncode: {}", proc.returncode)
```

Comment:
Looks like I meant these print statements to have a `format` to go with the `{}`. Let's either add format or remove curly braces. I realize you were probably just copying what was there but since in this part of the code right now let's get changed.

Reply:
Will fix in the AM. Have a good Sunday.